package.json

```diff
@@ -53,6 +54,14 @@
     ]
   },
   "jest": {
+    "coverageThreshold": {
+      "global": {
+        "branches": 100,
+        "functions": 100,
+        "lines": 100,
+        "statements": -10
+      }
+    },
```

Comment:
Do this match the CodeCov configuration?

Reply:
@camacho I think so. Currently `codecov` becomes an issue when the coverage drops, at the moment we have 100%.